loginserver/options.h

```diff
@@ -152,6 +163,7 @@ class Options {
 	bool        auto_link_accounts;
 	bool        update_insecure_passwords;
 	int         encryption_mode;
+	int			max_expansions;
```

Comment:
Replace the tab with spaces between the `int` and `max_expansions` here for alignment

Reply:
Done